### ISSUE TITLE
I got an error on 'roundup' command

### ISSUE THREAD

**Issue by minsookim1983** [wwood/finishm#39]
Hello,

I am trying to do 'roundup' with contigs, but I got an error that I attached.
[error.txt](https://github.com/wwood/finishm/files/458879/error.txt)
Please, give me any way to solve this problem.
Thank you.

Min-Soo Kim

**Comment by wwood** [comment]
Hi Min-Soo,

Hmm, there seems to be some problem running the velveth that comes with finishm, which I would guess is due to some issue with the input files. Would you mind running the following commands and reporting back the results please, from the same directory where finishm was run?

```
file d6f32d9d-7ba0-42ed-97ea-d79eb91abdae/msk2/lysogen_data2/bacteriome.merge/all.screened.mm10.fa.pair.R1.fq
file d6f32d9d-7ba0-42ed-97ea-d79eb91abdae/msk2/lysogen_data2/bacteriome.merge/all.screened.mm10.fa.pair.R2.fq

head d6f32d9d-7ba0-42ed-97ea-d79eb91abdae/msk2/lysogen_data2/bacteriome.merge/all.screened.mm10.fa.pair.R1.fq
head d6f32d9d-7ba0-42ed-97ea-d79eb91abdae/msk2/lysogen_data2/bacteriome.merge/all.screened.mm10.fa.pair.R2.fq
```

thanks.

**Comment by minsookim1983** [comment]
Here is the results of the commands that you gave me.
[error.txt](https://github.com/wwood/finishm/files/458946/error.txt)
As the results, something is wrong with the input files.

Min-Soo

**Comment by wwood** [comment]
It seems those files don't exist, or I'm not interpreting things correctly. How are you running finishm? i.e. what command line?

**Comment by minsookim1983** [comment]
I am sorry to bother you. I just found what was wrong with me. As you indicated, the input file path was wrong. I corrected it, and it is working now. Thank you.

**Comment by wwood** [comment]
Nice easy problem then, no worries.

**Comment by minsookim1983** [comment]
One thing more that I want to ask is that, Is there any way to set cpu number? 
I did not find any flags for cpu number in Usage. I checked issue #4, and I guess it is still not adopted to finishM.

**Comment by wwood** [comment]
Currently only the velvet step is multithreaded and in my experience much of that does not multithread. However, it can be useful by setting the environment variables as velvet describes in its manual (as in #4). I really should fix #4 though..
